src/main/java/org/opensearch/security/configuration/PrivilegesInterceptorImpl.java

```diff
@@ -95,9 +96,10 @@ private boolean isTenantAllowed(final ActionRequest request, final String action
      */
     @Override
     public ReplaceResult replaceDashboardsIndex(final ActionRequest request, final String action, final User user, final DynamicConfigModel config,
+                                                final TenancyConfigModel tenancyconfig,
                                                 final Resolved requestedResolved, final Map<String, Boolean> tenants) {
 
-        final boolean enabled = config.isDashboardsMultitenancyEnabled();//config.dynamic.kibana.multitenancy_enabled;
+        final boolean enabled = tenancyconfig.isDashboardsMultitenancyEnabled();
```

Comment:
nit: Camel Case

Reply:
Thanks @anijain-Amazon for finding this miss. Will update in next commit.

Reply:
With this change, would `config.dynamic.kibana.multitenancy_enabled` be ignored? Would this work in mixed clusters? Based on my understanding, security index is node-scoped. If there're 2 settings(the old and the new), could there be 2 different values(`false` and `true`) of multitenancy_enabled setting in a mixed cluster?

Reply:
@cliu123 going forward we will only use `config.dynamic.kibana.multitenancy_enabled` and this will get priority over `config.dynamic.kibana.multitenancy_enabled`.  

> If there're 2 settings(the old and the new), could there be 2 different values(false and true) of multitenancy_enabled setting in a mixed cluster?

In this case we will give priority to new settings.